frontend/src/components/Hashtags.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <div class="wrapper">
+    <div class="label">Najbolj uporabljeni hashtagi</div>
+    <div class="hashtagContainer">
+      <div v-for="hashtag in topHashtags" :key="hashtag" class="hashtag">
+        {{ hashtag }}
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { defineComponent } from 'vue'
+
+export default defineComponent({
+  props: {
+    hashtags: { type: Array, default: () => ({}), required: true },
```

Comment:
1. If you're feeling extra nice, maybe define the `Hashtag` type, import it in this component and do `Array as PropType<Hashtag>` (`PropType` can be imported from Vue). What do you generally use, React? Vue's TS support is quite janky in comparison.
2. I'd say `default` and `required` are mutually exclusive, maybe just remove the former.